providers/runtime.go

```diff
@@ -253,7 +253,7 @@ func (r *Runtime) Connect(req *plugin.ConnectReq) error {
 
 	r.Recording.EnsureAsset(r.Provider.Connection.Asset, r.Provider.Instance.ID, r.Provider.Connection.Id, asset.Connections[0])
 	r.schema.prioritizeIDs(BuiltinCoreID, r.Provider.Instance.ID)
-	r.schema.refresh()
+	r.schema.unsafeRefresh()
```

Comment:
same question about why this is safe without a lock

Reply:
same, this was an oversight